cv_next/app/feed/components/CVItemRSC.tsx

```diff
@@ -1,44 +1,53 @@
-import Image from "next/image"
+import Image, { StaticImageData } from "next/image"
 import Link from "next/link"
 import generateImageUrl from "@/helpers/imageURLHelper"
 import Categories from "@/types/models/categories"
-import { ClientCvModel } from "@/types/models/cv"
+import { useState } from "react"
+import blankCvImg from "../../../public/images/blackCv.png";
 interface CVCardProps {
-  cv: ClientCvModel
+  cv: CvModel
 }
 
 export default function CVItemRSC({ cv }: CVCardProps) {
   const categoryLink = `/feed?category=${Categories.category[
-    cv.categoryID
+    cv.category_id
   ].toLowerCase()}`
-  const imageUrl = generateImageUrl(cv.documentLink)
-  const formattedDate = new Date(cv.uploadDate).toLocaleDateString("en-US")
+  const imageUrl = generateImageUrl(cv.document_link)
+  const formattedDate = new Date(cv.created_at).toLocaleDateString("en-US")
+
+  const [imgSrc, setImgSrc] = useState<StaticImageData|string>(imageUrl)
+
+  const handleImgError = (e: any) => {
+    e.stopPropagation()
+    setImgSrc(blankCvImg)
+  }
 
   return (
     <>
       <Image
         width={500}
         height={500 * 1.4142}
         className="w-full rounded-lg p-2"
-        src={imageUrl}
+        src={imgSrc}
+        onError={handleImgError}
         alt="CV Preview"
         priority
       />
-      <div className="overlay gradient-blur-backdrop absolute bottom-0 flex h-1/6 w-full rounded-b-lg">
-        <div className="overlay absolute bottom-0 h-1/5 w-full rounded-b-lg bg-white p-6">
+      <div className="overlay backdrop-blur-[0.5px] gradient-blur-backdrop hover:via-transparent hover:backdrop-blur-none absolute bottom-0 flex h-full w-full rounded-lg transition">
+        <div className="overlay absolute bottom-0 h-1/5 w-full rounded-xl bg-transparent p-6">
           <div className="absolute bottom-0 left-0 mx-5 mb-2.5">
-            <div className="mr-2 inline text-xl font-bold text-neutral-700">
-              {cv.userID}
+            <div className="mr-2 inline text-xl font-bold text-black">
+              {cv.user_id}
             </div>
-            <p className="inline text-xs text-neutral-700/75">
+            <p className="inline text-xs text-neutral-400">
               {formattedDate}
             </p>
           </div>
           <Link
             href={categoryLink}
-            className="absolute bottom-0 right-0 mx-4 mb-2.5 rounded-full bg-neutral-200 px-3 py-1 text-sm font-semibold text-neutral-700 hover:underline"
+            className="absolute bottom-0 right-0 mx-4 mb-2.5 rounded-full bg-gray-700 px-3 py-1 text-sm font-semibold text-white hover:underline hover:bg-gray-400"
```

Comment:
General note to everything style-related: I don't know how exactly you're handling it now, but I think it'd be best if @PeroG28 will make a file with the necessary colors - optimally we'd have a fixed schema of colors that we know to use throughout the app.